README.md

```diff
@@ -292,6 +292,35 @@ auth0GuardianJS.start(function(err, transaction) {
   //...
 });
 ```
+### auth0GuardianJS.resume(options, transactionState, callback) 
+This continues a transaction saved by `transaction.serialize()`. The option parameter provides 
+the final user the opportunity to specify which kind of `client` to use. That been:
```

Comment:
`client` ?? what's client? later it's used as transport: `{ transport: 'pooling' }`

Something like this might sound better:
> The option parameter provides the final user the opportunity to specify which kind of `transport` to use, the options being: